examples/cs/comm/tls/Tls.cs

```diff
@@ -13,7 +13,8 @@ namespace Bond.Examples.Tls
 
     public static class Tls
     {
-        static readonly IPEndPoint serviceEndpoint = new IPEndPoint(IPAddress.Loopback, EpoxyTransport.DefaultSecurePort);
+        static readonly IPEndPoint serviceEndpointIpv4 = new IPEndPoint(IPAddress.Loopback, EpoxyTransport.DefaultSecurePort);
+        static readonly IPEndPoint serviceEndpointIpv6 = new IPEndPoint(IPAddress.IPv6Loopback, EpoxyTransport.DefaultSecurePort);
```

Comment:
I'm pretty sure that the IPv4 and IPv6 port spaces are distinct and can be bound independently. Let me validate.